public class com.novell.ldapchai.exception.ChaiRuntimeException extends java.lang.RuntimeException
{
public void <init>()
{
com.novell.ldapchai.exception.ChaiRuntimeException v0;
v0 := @this: com.novell.ldapchai.exception.ChaiRuntimeException;
specialinvoke v0.<java.lang.RuntimeException: void <init>()>();
return;
}
public void <init>(java.lang.String)
{
java.lang.String v1;
com.novell.ldapchai.exception.ChaiRuntimeException v0;
v0 := @this: com.novell.ldapchai.exception.ChaiRuntimeException;
v1 := @parameter0: java.lang.String;
specialinvoke v0.<java.lang.RuntimeException: void <init>(java.lang.String)>(v1);
return;
}
public void <init>(java.lang.String, java.lang.Throwable)
{
java.lang.Throwable v2;
java.lang.String v1;
com.novell.ldapchai.exception.ChaiRuntimeException v0;
v0 := @this: com.novell.ldapchai.exception.ChaiRuntimeException;
v1 := @parameter0: java.lang.String;
v2 := @parameter1: java.lang.Throwable;
specialinvoke v0.<java.lang.RuntimeException: void <init>(java.lang.String,java.lang.Throwable)>(v1, v2);
return;
}
}